react-native/README.md

```diff
@@ -94,13 +94,15 @@ See [the objC style guide](../osx/STYLEGUIDE.md)
 ### Javascript settings
 
 ```
-npm install -g standard
-npm install -g babel-eslint
 npm install -g flow
 ```
 
 See [the JS style guide](standardjs.com)
+We're extending standard to be more strict in some cases (see our .eslintrc)
+
 Visit [standardjs.com](http://standardjs.com/#text-editor-plugins) to find plugins for your editor of choice
+We're including local plugins which requires you to run eslint locally as well (see https://github.com/eslint/eslint/issues/1238)
```

Comment:
Please make sure your editor is doing the right thing. the vim thing is a little tricky